src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -237,6 +245,60 @@ function toJSON(inst : Instance | TextInstance) : ReactTestRendererNode {
   }
 }
 
+const propsWithoutChildren = (props) => {
+  /* eslint-disable no-unused-vars */
+  var { children, ...otherProps } = props;
+  /* eslint-enable */
+  return otherProps;
+};
+
+function nodeAndSiblingsArray(nodeWithSibling) {
+  var node = nodeWithSibling;
+  var array = [node];
+  while (node.sibling !== null) {
+    array.push(node.sibling);
+    node = node.sibling;
+  }
+  return array;
+}
+
+function toTree(node) {
+  switch (node.tag) {
+    case HostRoot: // 3
+      return toTree(node.progressedChild);
```

Comment:
The problem is that you're looking at the wrong root. There's two. We might also want to switch which one createContainer returns.

For now, use `stateNode` to store the "root".

```js
var root = TestRenderer.createContainer(container).stateNode;
```

This will not be a Fiber object so every time you use it you need to use `root.current` to get the current Fiber.

There are two Fibers per root and this will ensure that you get the right one.